docs/advanced-table-feature.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Advanced table feature"
```

Comment:
Is this advanced table feature? Partition should be basic feature for a table for me.
What's the advanced feature?
I think there are multiple tables transactions, auto optimizer, CBO etc.